R/pkg/inst/tests/testthat/test_context.R

```diff
@@ -72,6 +72,20 @@ test_that("repeatedly starting and stopping SparkSession", {
   }
 })
 
+test_that("Default warehouse dir should be set to tempdir", {
+  sparkR.session.stop()
+  sparkR.session(enableHiveSupport = FALSE)
+
+  # Create a temporary table
+  sql("CREATE TABLE people_warehouse_test")
+  # spark-warehouse should be written only tempdir() and not current working directory
+  res <- list.files(path = ".", pattern = ".*spark-warehouse.*",
```

Comment:
You are right.
In this case we are specifically looking for spark-warehouse; I guess I was referring to a general check to make sure going forward the list of files before running anything in the package should  == list of files after running anything